applications/openshift/api-server/api_server_encryption_provider_cipher/rule.yml

```diff
@@ -0,0 +1,39 @@
+documentation_complete: true
+
+prodtype: ocp4
+
+title: 'Configure the Encryption Provider Cipher'
+
+description: |-
+    To configure OpenShift to use the <tt>aescbc</tt> encryption provider,
+    follow the OpenShift documentation and configure a
+    <tt>EncryptionConfig</tt> file <tt>/etc/origin/master/encryption-config.yaml</tt>.
+    In this file, choose <tt>aescbc</tt> as the encryption provider:
+    <pre>kind: EncryptionConfig
+    apiVersion: v1
+    resources:
+      - resources:
+        - secrets
```

Comment:
Are these lines related to this configuration setting?

`````
      - resources:
        - secrets
`````

If not, should they be removed from the example? Some users may literally interpret these lines and assume they must be present and blank for this check.

Reply:
@shawndwells can't say yet. The example will be changing for JSON.